content/influxdb/cloud/visualize-data/annotations.md

```diff
@@ -21,25 +21,24 @@ Add annotations to your dashboards to provide useful, contextual information abo
 Annotations may be useful to highlight operations or anomalies for your team to reference.
 {{% /note %}}
 
-{{< youtube 5NEplCesNAc >}}
+<!-- {{< youtube 5NEplCesNAc >}} --->
 
 ## Create an annotation
 
 1. In the navigation menu on the left, select **Boards** (**Dashboards**).
 
     {{< nav-icon "dashboards" >}}
 
-2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/).
-3. Click **Annotations**. The **Show Annotations** option is selected by default.
-4. Select the **Enable 1-Click Annotations** option, and then in any dashboard cell, click the point in time to add the annotation.
-  {{% note %}}
-**Tip:** To move the annotation outside of the selected graph time range, first create the annotation, and then [edit the annotation](#edit-an-annotation) to update the timestamp as needed.
-  {{% /note %}}
-5. On the **Add Annotation** page:
+2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/). The **Annotations** option is selected by default.
+3. In a dashboard cell, doing one of the following:
 
-   1. Verify the start time, and update if needed.
-   2. Enter a message (maximum of 255 characters) to associate with the selected start time.
-   3. Click **Save Annotation**. The annotation appears as a dotted line at the specified start time and includes details associated with the selected point.
+   - To add an annotation to a single point in time, press Shift and click the time.
```

Comment:
should this be something more like "click the graph at the time you want to annotate" ?

Reply:
```suggestion
   - Press Shift and click the graph at the time you want to annotate.
```